src/clients/dotnet/TigerBeetle.Tests/IntegrationTests.cs

```diff
@@ -1141,18 +1173,6 @@ public void Dispose()
 
         private void CleanUp()
         {
-            try
-            {
-                if (Process.GetProcessesByName(TB_EXE) is Process[] runningList)
-                {
-                    foreach (var runningProcess in runningList)
-                    {
-                        runningProcess.Kill();
-                    }
-                }
-            }
-            catch { }

```

Comment:
Humm... I remembered why this code is there: In the earlier days of the dotnet client it was common to the client to segfault leaving the server running... so I put it as a band-aid to avoid manually killing the process each time but forgot to remove 🙈

Reply:
I am wondering what _should_ cleanup stray processes? My guess is that actions are run as a separate process group, with a shell attached to a pty, so hopefully everything is cleaned up across jobs. 

But yeah, this code is why I suggest "tigerbeetle exits if stdin is closed"